lib/catalog/api/minion/logging.rb

```diff
@@ -8,7 +8,12 @@ class << self
       end
 
       def self.logger
-        @logger ||= ManageIQ::Loggers::CloudWatch.new
+        if ENV["LOG_TO_STDOUT"].present?
+          @logger ||= ManageIQ::Loggers::Base.new(STDOUT)
+        else
+          @logger ||= ManageIQ::Loggers::CloudWatch.new
+        end
+
```

Comment:
do we need to enable the messaging gem to use our logs using
```
ManageIQ::Messaging.logger = @logger
```
here

Reply:
I'm not sure.  I was able to verify that adding the code to the generic ManageIQ::Loggers class did work as expected in my local environment. I'm willing to try the `ManageIQ::Messaging.logger = @logger` but wasn't sure at all how it would work.